src/test/java/uk/ac/ebi/atlas/download/FileDownloadControllerTest.java

```diff
@@ -122,12 +123,27 @@ public void testBadAccessions() {
     @Test
     public void testEmptyAccessions() {
 
```

Comment:
Be judicious (but mostly consistent) about the use of blank lines. Have a look at your code overall to remove any blank line that doesn’t improve code readability.